tests/_support/AcceptanceTester.php

```diff
@@ -227,4 +225,30 @@ public function iShouldSeeAnImageIsLinkedTo($image, $expectedUrl)
         $actualUrl = $this->grabAttributeFrom("//img[@src='$image']/parent::*", "href");
         $this->assertEquals($expectedUrl, $actualUrl);
     }
+
+    /**
+     * @When I sign in gigadb website as :user
+     */
+    public function iSignInGigaDBWebsiteAsUser($user)
```

Comment:
This function is not required because you can use `iSignInAsAUser()` in `AuthorSteps.php`?